package.json

```diff
@@ -41,45 +41,47 @@
     "build:assets": "npm-run-all assets:clean -p js sass",
     "build": "run-s build:assets build:site eleventy"
   },
+  "dependencies": {},
   "devDependencies": {
-    "@11ty/eleventy": "^0.9.0",
-    "@11ty/eleventy-plugin-syntaxhighlight": "^2.0.3",
-    "@babel/core": "^7.6.4",
-    "@babel/preset-env": "^7.6.3",
-    "accoutrement": "^2.1.2",
+    "@11ty/eleventy": "^0.10.0",
```

Comment:
@mirisuzanne It's possible this upgrade includes some features we might want to use? Not sure, but take a look at: https://github.com/11ty/eleventy/releases/tag/v0.10.0

Reply:
The only update that's particularly interesting to me is converting our global/folder `json` data files to `yaml` for consistency with file data: `eleventyConfig.addDataExtension("yaml", contents => yaml.safeLoad(contents));` Not sure if it's that simple, or not. We can either play with it in this branch, or make a story to  look at it later.

Reply:
Added a story: https://trello.com/c/vhNKI3YC